metron-analytics/metron-profiler/src/test/config/zookeeper/readme-example-1/profiler.json

```diff
@@ -0,0 +1,17 @@
+{
+  "inputTopic": "indexing",
+  "profiles": [
+    {
+      "profile": "example1",
+      "foreach": "ip_src_addr",
+      "onlyif": "protocol == 'HTTP'",
+      "init": {
+        "total_bytes": 0.0
```

Comment:
Can init take stellar statements?  There will be situations where we'll want to init via stellar (e.g. bloom filter with specific parameters).  If not, we should make it accept it.